assets/src/phoenix/app.ts

```diff
@@ -155,6 +155,12 @@ window.toggleAudio = (element: HTMLAudioElement) => {
   }
 };
 
+window.addEventListener('phx:js-exec', ({ detail }: any) => {
+  document.querySelectorAll(detail.to).forEach((el) => {
+    liveSocket.execJS(el, el.getAttribute(detail.attr));
+  });
+});
```

Comment:
❤️ this implementation! we will reuse it for future features for sure!

Reply:
Thanks. I came up with it by myself 👀 😂